src/reaktoro_pse/tutorials/integration_with_ro.ipynb

```diff
@@ -702,81 +702,17 @@
-      "| :: Ca+2         | 9.6310e-03 |  mol |\n",
-      "| :: Cl-          | 4.6272e-01 |  mol |\n",
-      "| :: HCO3-        | 2.1284e-03 |  mol |\n",
-      "| :: SO4-2        | 2.6610e-02 |  mol |\n",
-      "| :: HSO4-        | 1.0015e-08 |  mol |\n",
-      "| :: Mg+2         | 5.0088e-02 |  mol |\n",
-      "| :: K+           | 9.3791e-03 |  mol |\n",
-      "| :: MgCO3        | 2.0220e-05 |  mol |\n",
-      "| :: MgOH+        | 6.1448e-07 |  mol |\n",
-      "| :: Na+          | 4.4310e-01 |  mol |\n",
-      "| :: OH-          | 5.8203e-07 |  mol |\n",
-      "+-----------------+------------+------+\n",
-      "2024-10-28 19:08:48 [INFO] idaes.reaktoro_pse.reaktoro_block: -----Displaying information for property block ------\n",
-      "2024-10-28 19:08:48 [INFO] idaes.reaktoro_pse.reaktoro_block: +-----------------+------------+------+\n",
-      "| Property        |      Value | Unit |\n",
-      "+-----------------+------------+------+\n",
-      "| Temperature     |   298.0000 |    K |\n",
-      "| Pressure        |     1.0132 |  bar |\n",
-      "| Charge:         | 5.3807e-02 |  mol |\n",
-      "| Element Amount: |            |      |\n",
-      "| :: H            | 1.0713e+02 |  mol |\n",
-      "| :: C            | 2.2512e-03 |  mol |\n",
-      "| :: O            | 5.3677e+01 |  mol |\n",
-      "| :: Na           | 4.4310e-01 |  mol |\n",
-      "| :: Mg           | 5.0108e-02 |  mol |\n",
-      "| :: S            | 2.6610e-02 |  mol |\n",
-      "| :: Cl           | 4.6272e-01 |  mol |\n",
-      "| :: K            | 9.3791e-03 |  mol |\n",
-      "| :: Ca           | 9.6310e-03 |  mol |\n",
-      "| Species Amount: |            |      |\n",
-      "| :: H+           | 3.6980e-08 |  mol |\n",
-      "| :: H2O          | 5.3563e+01 |  mol |\n",
-      "| :: CO3-2        | 2.1445e-05 |  mol |\n",
-      "| :: CO2          | 8.1147e-05 |  mol |\n",
-      "| :: Ca+2         | 9.6310e-03 |  mol |\n",
-      "| :: Cl-          | 4.6272e-01 |  mol |\n",
-      "| :: HCO3-        | 2.1284e-03 |  mol |\n",
-      "| :: SO4-2        | 2.6610e-02 |  mol |\n",
-      "| :: HSO4-        | 1.0015e-08 |  mol |\n",
-      "| :: Mg+2         | 5.0088e-02 |  mol |\n",
-      "| :: K+           | 9.3791e-03 |  mol |\n",
-      "| :: MgCO3        | 2.0220e-05 |  mol |\n",
-      "| :: MgOH+        | 6.1448e-07 |  mol |\n",
-      "| :: Na+          | 4.4310e-01 |  mol |\n",
-      "| :: OH-          | 5.8203e-07 |  mol |\n",
-      "+-----------------+------------+------+\n"
+      "2025-03-25 14:07:53 [INFO] idaes.reaktoro_pse.reaktoro_block: -----Displaying information for speciation block ------\n",
+      "<reaktoro_pse.core.reaktoro_state.ReaktoroState object at 0x000001D825FE2910>\n",
+      "2025-03-25 14:07:53 [INFO] idaes.reaktoro_pse.reaktoro_block: -----Displaying information for property block ------\n",
+      "<reaktoro_pse.core.reaktoro_state.ReaktoroState object at 0x000001D8253B9A50>\n"
```

Comment:
Same thing about non-human-readable `ReaktoroState` repr in the changed version.